DQMServices/Components/plugins/DQMGenericClient.cc

```diff
@@ -656,6 +656,11 @@ void DQMGenericClient::makeAllPlots(DQMStore::IBooker& ibooker, DQMStore::IGette
                         efficOption->type,
                         efficOption->isProfile);
     }
+    if (makeGlobalEffPlot_) {
+      ME* globalEfficME = igetter.get(dirName + "/globalEfficiencies");
+      if (globalEfficME)
+        globalEfficME->getTH1F()->LabelsDeflate("X");
+    }
```

Comment:
This will affect **all** users of `DQMGenericClient`. Would you be so kind of explain what this change will do?

Reply:
From [ROOT documentatoin](https://root.cern.ch/doc/master/classTH1.html#alpha):
When using the `Fill` function with one of the arguments being a string, new labels are added by doubling the current number of bins in case one label does not exist yet. When the Filling is terminated, it is possible to trim the number of bins to match the number of active labels by calling `LabelsDeflate()`.

Without calling `LabelsDeflate()`:
https://lecriste.web.cern.ch/lecriste/HGCAL/ValidationMetrics/purities_sep/plots1_ticlSimTracksters/Efficiencies/globalEfficiencies.png